ros/angel_system_nodes/angel_system_nodes/object_detector_with_descriptors.py

```diff
@@ -334,11 +318,9 @@ def postprocess_detections(self, rois, cls_prob, pooled_feat, im_scales):
             else:
                 objects = torch.argmax(scores[keep_boxes][:,1:], dim=1)
                 box_dets = np.zeros((len(keep_boxes), 4))
-                boxes = pred_boxes[keep_boxes.cpu()]
+                boxes = filled_boxs[keep_boxes].cpu()
                 for i in range(len(keep_boxes)):
-                    kind = objects[i]+1
-                    bbox = boxes[i, kind * 4: (kind + 1) * 4]
-                    box_dets[i] = bbox
+                    box_dets[i] = boxes[i]
```

Comment:
I think these three lines (321-323) can be collapsed into just:
```
box_dets = filled_boxs[keep_boxes].cpu().numpy()
```
The looping in python here is likely costly.